modules/installation-initializing.adoc

```diff
@@ -548,6 +558,18 @@ vpcSubnets: <vpcSubnet>
 +
 For `platform.powervs.vpcName`, specify the name for the existing {ibm-cloud-name}. For `platform.powervs.vpcSubnets`, specify the existing subnets.
 endif::ibm-power-vs+restricted[]
+ifdef::ibm-cloud+restricted[]
+.. Define the network and subnets for the VPC to install the cluster in under the parent `platform.ibmcloud` field:
```

Comment:
```suggestion
.. Define the VPC network and subnets for the cluster to use under the parent `platform.ibmcloud` field:
```
Tried rewording this because "in under" felt awkward.

Reply:
Yeah. This is language that is used for every restricted installation assembly. I agree it reads a little awkward, but I am not sure if your suggestion changes the technical accuracy. I am inclined to leave this one alone.